lnwallet/channel_test.go

```diff
@@ -129,7 +129,7 @@ func testAddSettleWorkflow(t *testing.T, tweakless bool,
 	// This signature will cover the HTLC, since Bob will first send the
 	// revocation just created. The revocation also acks every received
 	// HTLC up to the point where Alice sent here signature.
-	bobNewCommit, err := bobChannel.SignNextCommitment()
+	bobNewCommit, err := bobChannel.SignNextCommitment(nil)
```

Comment:
The `nil` looks slightly weird to me... Probably because it hides what type it should be.
We could just do `make(chan struct{})` to make it a bit more explicit (but with the same behavior).
Slightly longer so might cause the diff to be even larger, so non-blocking and only if you agree.

Reply:
Good point, having the type explicit in the tests will def be more readable later on.